.github/workflows/tests.yml

```
@@ -0,0 +1,54 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - "*"
+  push:
+    branches:
+      - master
```

The `master` branch was renamed to `main` in this repo
```suggestion
      - main
```